Assignment-RegistrationForm/src/UpdationPage/editFormStyle.css

```diff
@@ -0,0 +1,98 @@
+body {
+  font-family: Arial, Helvetica, sans-serif;
+  background-image: linear-gradient(45deg, #4158a6, #1eb8cf);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+h1 {
+  background: -webkit-linear-gradient(45deg, #4158a6, #1eb8cf);
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+}
+
+.form-panel {
+  background-color: #fff;
+  padding: 20px;
+  border-radius: 20px;
+  box-shadow: 0px 0px 20px rgb(0, 0, 0);
+  width: 400px;
+  min-height: 520px;
+}
+
+.form-panel:hover {
+  box-shadow: 0px 0px 30px rgb(0, 0, 0);
+  transform: translateY(-3px);
+}
+
+.elements {
+  margin-bottom: 15px;
+  display: flex;
+  align-items: center;
+}
+
+label {
+  width: 100px;
+  margin: 20px 10px 0 0;
+  font-weight: 600;
+  color: #0655bd;
+}
+
+input,
+textarea {
+  padding: 10px;
+  border: 1px solid #01a7e9;
+  margin-top: 10px;
+  border-radius: 6px;
+  background-color: #e1efff;
+  width: 300px;
+}
+
+.error-display {
+  margin: 0 0 10px 110px;
+  display: flex;
+}
+
+button:active {
+  transform: translateY(2px);
+}
+
+h1 {
+  text-align: center;
+  margin-bottom: 20px;
+  font-size: 35px;
+}
+
+#btn {
+  border: 1px solid #797777;
+  background-color: #a8a7a7;
+  border-radius: 4px;
+  color: #1d1c1c;
+  padding: 5px 3px 5px 2px;
+  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
+  letter-spacing: 4px;
+  font-weight: bolder;
+  opacity: 0.3;
+  width: 100%;
+  margin-top: 15px;
+}
+
+#btn:enabled {
+  background-image: linear-gradient(45deg, #4158a6, #1b9db1);
+  opacity: 1;
+  color: white;
+  cursor: pointer;
+}
+
+#btn:enabled:hover {
+  background-image: linear-gradient(45deg, #3c5196, #178c9e);
+  transition: all 350ms ease-in-out;
+}
+
+.error-display {
+  color: red;
```

Comment:
error-display needs to be combined together. Remove this and add the styles to the top class itself.